blog/steering/[BLOG]-toc-2021-election-announcement.md

```diff
@@ -0,0 +1,42 @@
+It’s time to kick off the Knative Technical Oversight Committee (TOC) elections
```

Comment:
Wonder if we should make the TOC link here instead of L5 below?

Reply:
I'll actually do both with this link going to the main TOC page and the other link going specifically to the charter section.